src/app/components/Notification/NotificationContainer.tsx

```diff
@@ -0,0 +1,34 @@
+import { ToastContainer } from "react-toastify";
+import { twJoin } from "tailwind-merge";
+import { useWindowSize } from "usehooks-ts";
+
+const commonClassName = "relative overflow-hidden rounded-lg px-4 py-3 md:p-3";
+
+const BG_COLOR = {
+  success:
+    "dark:bg-[linear-gradient(0deg,rgba(73,177,73,0.1),rgba(73,177,73,0.1))] dark:bg-[#191919] bg-[linear-gradient(0deg,rgba(73,177,73,0.05),rgba(73,177,73,0.05))] bg-[#FFFFFF]",
+  warning:
+    "bg-[linear-gradient(0deg,rgba(197,136,45,0.05),rgba(197,136,45,0.05))] bg-[#FFFFFF] dark:bg-[linear-gradient(0deg,rgba(197,136,45,0.1),rgba(197,136,45,0.1))] dark:bg-[#191919]",
+  error:
+    "bg-[linear-gradient(0deg,rgba(221,100,100,0.05),rgba(221,100,100,0.05))] bg-[#FFFFFF] dark:bg-[linear-gradient(0deg,rgba(221,100,100,0.1),rgba(221,100,100,0.1))] dark:bg-[#191919]",
+  info: "bg-[#FFFFFF] dark:bg-[#191919]",
+  default: "bg-[#FFFFFF] dark:bg-[#191919]",
+} as const;
+
+export const NotificationContainer = () => {
+  const { width: screenWidth } = useWindowSize();
```

Comment:
we need to use `https://usehooks-ts.com/react-hook/use-media-query`, export the screen sizes somewhere so that they match the `tailwind.config.ts`

Reply:
I have extracted the screen config from `tailwind.config.ts` so that it can be shared between tailwind so our code. Let me know what do you think.